tools/ros1_dependencies.repos.melodic

```diff
@@ -1,12 +1,4 @@
 repositories:
-  turtlebot3_simulations:
```

Comment:
These have rosdep keys, today, no package actually depends on these for navigation2. If you'd like to include these, they should not be done in CI (which as of this PR we're not building ROS1 deps with CI)

Reply:
Issue is here